ws/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM openjdk:14.0.1-slim
 MAINTAINER Andrej Sajenko <Andrej.Sajenko@mni.thm.de>
 
 ADD build/install/ws /usr/local/ws
-ADD src/main/resources/init.sql /usr/local/ws
+ADD src/main/resources/fbs.sql /usr/local/ws
```

Comment:
If the resources folder already contains the fbs.sql file, the file will be part of the compiled application.
Why do we need to add the sql file into the container?